data-pipeline/tests/test_fetch_info.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright 2021-Present Datadog, Inc. https://www.datadoghq.com/
+// SPDX-License-Identifier: Apache-2.0
+
+#[cfg(test)]
+mod require_test_agent {
```

Comment:
These tests don't seem to pass yet. Are you still waiting on changes from the test-agent to be deployed?